# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../k3b/uic-issue.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur k3b-0.12.4a.orig/src/option/base_k3bdivxoptiontab.ui k3b-0.12.4a/src/option/base_k3bdivxoptiontab.ui
--- k3b-0.12.4a.orig/src/option/base_k3bdivxoptiontab.ui	2005-09-10 16:30:45.000000000 +0300
+++ k3b-0.12.4a/src/option/base_k3bdivxoptiontab.ui	2005-10-07 13:21:47.000000000 +0300
@@ -326,7 +326,6 @@
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kdoublenuminput.h</includehint>
     <includehint>knuminput.h</includehint>
     <includehint>knuminput.h</includehint>
     <includehint>knuminput.h</includehint>
